### ISSUE TITLE
yank --store does not work

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#297]
```
mski1776:abl-imatinib-implicit choderaj$ yank status  --store=output
File output/solvent.nc not found.
```

**Comment by luirink** [comment]
I think it is not  --store that is not working. In analyze.py in print_status, the suffix ('implicit' or 'explicit') is not connected to the fullpath. 

I suggest to change 388-392 in analyze.py to:

    # Process each netcdf file.
    phases = ['solvent', 'complex']
    suffixes = ['explicit', 'implicit']
    for phase in phases:
        for suffix in suffixes:
        # Construct full path to NetCDF file.
            fullpath = os.path.join(store_directory, '%s-%s.nc' % (phase, suffix))

But also to change the return False case, since it will be the case that implicit or explicit does not exist.

**Comment by luirink** [comment]
phases = ['solvent', 'complex']
    suffixes = ['explicit', 'implicit']
    a=0
    for phase in phases:
        for suffix in suffixes:
        # Construct full path to NetCDF file.
            fullpath = os.path.join(store_directory, '%s-%s.nc' % (phase, suffix))

        # Check that the file exists.
            if (not os.path.exists(fullpath)):
                a+=1
                if a==4:
                    # Report failure.
                    logger.info("No nc-files found.")
                    logger.info("Check to make sure the right directory was specified, and 'yank setup' has been run.")
                    return False
                else:
                    continue


Ugly but easy way.. Works for me!

**Comment by andrrizzi** [comment]
I think in general we should not rely on hardcoded phase names like `complex-explicit`, especially since we want to extend it to solvation free energies.

**Comment by jchodera** [comment]
I'd love to find a better solution for this. Should there be a master index
listing which NetCDF stores are present and how they should be analyzed to
give the free energy estimates of interest?

**Comment by andrrizzi** [comment]
This sounds like a good solution!

An alternative could be just saving into the phase `.nc` file the "sign" of the in the thermodynamic cycle arrow so that we know what we need to sum and what to subtract.

**Comment by andrrizzi** [comment]
I think this was addressed by #420 . Feel free to reopen if this is a problem.
